packages/outputarea/src/model.ts

```diff
@@ -255,13 +264,21 @@ export class OutputAreaModel implements IOutputAreaModel {
     return this._add(output);
   }
 
+  /**
+   * Remove an output at a given index.
+   */
+  remove(index: number): void {
+    this.list.get(index).dispose();
+    this.list.remove(index);
```

Comment:
`remove` returns the item a more efficient pattern is:

```suggestion
    this.list.remove(index)?.dispose();
```

Reply:
Thanks, done in https://github.com/jupyterlab/jupyterlab/pull/16498/commits/0bd367e40639658d4d1dc29ed41b3d5cb64a3a9b.